mix.exs

```diff
@@ -35,13 +35,10 @@ defmodule Kino.MixProject do
 
   defp docs do
     [
-      main: "readme",
```

Comment:
Will miss the logo :see_no_evil: Should we have a text-less version of the logo in the sidebar?

Reply:
Yes, let's do that instead :) Can you please do so in a separate commit?

Reply:
Surething!